vendor/sigs.k8s.io/kustomize/api/types/kustomization.go

```diff
@@ -59,12 +59,14 @@ type Kustomization struct {
 	// CommonAnnotations to add to all objects.
 	CommonAnnotations map[string]string `json:"commonAnnotations,omitempty" yaml:"commonAnnotations,omitempty"`
 
+	// Deprecated: Use the Patches field instead, which provides a superset of the functionality of PatchesStrategicMerge.
```

Comment:
for all of these deprecated indicators, does the previous functionality still work, but with a warning now?